# This is a single comms write

# Destination for the read or write
uint8 MSP430=0
uint8 IMU=1
uint8 device

# addr is the address of the desired register
uint8 addr
uint8[] data